rasa/core/test.py

```diff
@@ -617,24 +635,33 @@ def _collect_action_executed_predictions(
             # Even if the prediction is also wrong, we don't have to undo the emulation
             # of the action rejection as we know that the user explicitly specified
             # that something else than the form was supposed to run.
-            predicted = action.name()
+            predicted_action = action.name()
 
-    action_executed_eval_store.add_to_store(
-        action_predictions=[predicted], action_targets=[gold]
+    should_ignore_prediction = action_executed_eval_store.should_ignore_prediction(
```

Comment:
I think the name `should_ignore_prediction` was confusing me before, as we do still compare the prediction. Maybe a name like `predicted_warning` would be clearer?